src/main/java/de/themorpheus/edu/taskservice/controller/solution/ImageSolutionController.java

```diff
@@ -69,9 +74,13 @@
 
 		ImageSolutionModel imageSolution = optionalImageSolution.get();
 		imageSolution.setUrl(dto.getUrl());
-		return ControllerResult.of(this.imageSolutionRepository.save(imageSolution));
+		imageSolution = this.imageSolutionRepository.save(imageSolution);
+
+		return ControllerResult.of(
+				new UpdateImageSolutionResponseDTO(imageSolution.getImageSolutionId(), imageSolution.getUrl()));
 	}
 
+	@Transactional
```

Comment:
Why?

Reply:
because hibernates needs to know if i call a mehtod where a model is used